src/Adapter/Security/Admin.php

```diff
@@ -77,6 +89,13 @@ public function __construct(LegacyContext $context, TokenStorageInterface $secur
      */
     public function onKernelRequest(GetResponseEvent $event)
     {
+        if (
```

Comment:
If I understand this, the purpose is to ignore the current security layer when the access token controller is called? Isn't there another solution instead of modifying this class and forcing a new parameter in its controller?

Maybe the event listener could be disabled when we are in the API context, that's one idea but the goal would be to handle this via configuration which would be less invasive than modifying the code directly I guess (not sure it's possible though 😅)

Reply:
Maybe we could create a new `api` env for the kernel, this would allow us to have a dedicated `config_api.yml` config file to override disable or change any configuration we need (maybe the league bundle would only be needed in this environment btw).

Of course we could then have some finer distinctions `config_api_dev.yml` and `config_api_prod.yml`